content/sensu-go/5.15/api/overview.md

```diff
@@ -135,6 +135,93 @@ http://127.0.0.1:8080/api/core/v2/namespaces/default/events
 If your token expires, you should see a 401 Unauthorized response from the API.
 To regenerate a valid access token, first run any sensuctl command (like `sensuctl event list`) then repeat step 2.
 
+### Authenticate with the API key feature
+
+The Sensu API key feature (core/v2.APIKey) is a persistent UUID that maps to a stored Sensu username. API keys are cluster-wide resources, so only cluster admins can grant, view, and revoke them.
```

Comment:
Present both here and in the guide below, I believe this description needs to be fleshed out some. Specifically, I think the description should answer the question "Why would I want to use API keys instead of access tokens described above?"

From my perspective there are multiple advantages:
* Check plugins, handler plugins and other code can integrate with the Sensu API without implementing the logic required to authenticate via the `/auth` API endpoint and periodically refresh the access token
* Username + password doesn't need to be provided in check or handler definitions (this is a big deal for a lot of security-conscious teams)
* API keys can be created/revoked without changing the underlying user's password
* Conversely (I expect that) API keys continue to work even if the user's password is changed.